app/src/main/java/com/yeonae/chamelezone/view/course/PlaceChoiceFragment.kt

```diff
@@ -46,14 +61,26 @@ class PlaceChoiceFragment : Fragment() {
 
         }
 
-        btn_back.setOnClickListener {
+        btn_cancel.setOnClickListener {
             requireActivity().onBackPressed()
         }
+
     }
 
     private fun setAdapter() {
         recycler_place_choice.layoutManager = LinearLayoutManager(context)
         recycler_place_choice.adapter = placeChoiceRvAdapter
     }
 
+    companion object {
+        fun newInstance(
+            visible: String
+        ) = PlaceChoiceFragment().apply {
+            arguments = Bundle().apply {
+                putString("visible", visible)
```

Comment:
```suggestion
                putString(VISIBLE, visible)
```
상수로 관리해두면 나중에 알아보기 편할거에요

Reply:
수정했습니다